pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
@palewire not sure what you are doing, but this has nothing to do with the doc-string. It doesn't matter if it *only* defined here, this NEEDS different parameters for klass. Otherwise this is not custom to the type (Series or DataFrame). 

Further it is impossible to actually see what you changed.